src/components/frame/v5/pages/MembersPage/hooks.ts

```diff
@@ -1,6 +1,13 @@
 import { usePopperTooltip } from 'react-popper-tooltip';
+import { useGetMembersForColonyQuery } from '~gql';
+import { useColonyContext } from '~hooks';
```

Comment:
Add a blank space, I think a good way is to keep separated by the blank line imports packages and other imports like components, hooks, types, styles, etc.